hs_swat_modelinstance/forms.py

```diff
@@ -0,0 +1,341 @@
+    has_title_drainage = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+    has_point_source = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+    has_fertilizer = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+    has_tillage_operation = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+    has_inlet_of_draining_watershed = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+    has_irrigation_operation = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+
+
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, **kwargs):
+        super(SWATModelParametersForm, self).__init__(*args, **kwargs)
+        self.helper = SWATModelParametersFormHelper(allow_edit, res_short_id, element_id, element_name='SWATModelParameters')
+        self.fields['has_crop_rotation'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+        self.fields['has_title_drainage'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+        self.fields['has_point_source'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+        self.fields['has_fertilizer'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+        self.fields['has_tillage_operation'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+        self.fields['has_inlet_of_draining_watershed'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+        self.fields['has_irrigation_operation'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+
+    class Meta:
+        model = SWATModelParameters
+        fields = ['has_crop_rotation',
+                  'has_title_drainage',
+                  'has_point_source',
+                  'has_fertilizer',
+                  'has_tillage_operation',
+                  'has_inlet_of_draining_watershed',
+                  'has_irrigation_operation',
+                  'has_other_parameters']
+        exclude = ['content_object']
+
+class SWATModelParametersValidationForm(forms.Form):
+    has_crop_rotation = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
+    has_title_drainage = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
+    has_point_source = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
+    has_fertilizer = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
+    has_tillage_operation = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
+    has_inlet_of_draining_watershed = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
+    has_irrigation_operation = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), required=False)
```

Comment:
This YES/NO pair is repeated a lot. Could probably pull that out into a constant.